WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -58,18 +58,14 @@ import Combine
         }
     }
 
-    lazy var syncContext: NSManagedObjectContext = {
-        return ContextManager.sharedInstance().newDerivedContext()
-    }()
-
-    lazy var service: ReaderPostService = {
-        return ReaderPostService(managedObjectContext: syncContext)
-    }()
+    private var coreDataStack: CoreDataStack {
+        ContextManager.shared
+    }
 
     /// An alias for the apps's main context – temporarily replaces  `newMainContextChildContext` until we have `NSPersistentContainer` support
     ///
     private var viewContext: NSManagedObjectContext {
```

Comment:
Do you think it would be worth renaming this to `mainContext`, just so it's clear in the code what context we are using?

At the same time, the consumer code shouldn't really care where the context comes from as long as it's valid. So as long as the name clearly shows the objects is an `NSManagedObjectContext` instance, we're good. 🤷‍♂️

Reply:
My assumption is "view context" is the preferred term, given it's what `NSPersistentContainer` uses to represent a context that's associated with the main thread. The term "main context" is only used in this codebase here and predates `NSPersistentContainer`.